tests/functional/test_override_database/test_override_database.py

```diff
@@ -31,6 +31,10 @@ def project_config_update(self):
             }
         }
 
+    def delete_alt_database_relation(self, project):
```

Comment:
If this is what we always want to do, and ensure that it runs regardless of the success of the unit test, can we run it as a teardown method?

Reply:
My initial design was for it to be a `fixture` or called a `tearDown` method but felt it might distract people in future troubleshooting cases as this method is only used to cover the small case that our functional tests default method of `drop_test_schema` misses which is renames or pointing to different projects from our default relation information. this really only happens in a very few tests this one, and `tests_aliases` being a notable.

the `fixture` didn't seem to trigger properly and would leave our hanging artifacts in the alt bigquery project, this version seems to work. so just figured to explicitly name the method based on what its needed for

Reply:
Ah I see. And this is where I'm lacking the context. My spidey sense is saying that we should look into the few cases where it doesn't work as expected then. But I agree, if it works most of the time, we should not run this as a teardown for all cases.

Reply:
I'd still prefer to see this as function 😁 It's a pretty quick thing to add a fixture to the class, and then, you could call it within the function definition of your test case wherever this behavior should be forced. See the `yield` statement in the fixtures docs -- Chenyu turned me onto this. I suspect this will appear in other tests and having it so easily accessible would be rad to the max

Reply:
played around and was able to get a fixture built out